examples/principal-multiple/principal-multiple.py

```diff
@@ -0,0 +1,35 @@
+import iam_floyd as statement
+import importlib
+import os
+import sys
+import inspect
+
+currentdir = os.path.dirname(os.path.abspath(
+    inspect.getfile(inspect.currentframe())))
+helperDir = '%s/../../helper/python' % currentdir
+sys.path.insert(0, helperDir)
+
+test = importlib.import_module('python_test')
+out = getattr(test, 'out')
+deploy = getattr(test, 'deploy')
+
+
+def statements():
+    # doc-start
+    s1 = statement.Sts() \
+                .allow() \
+                .to_assume_role() \
+                .for_account('1234567890', '0987654321')
```

Comment:
Thanks for implementing this!  Would the Python also accept a `list` or a `tuple`?  I guess maybe with unpacking?  Like `.for_account(*my_list_of_account_ids)` ?

Reply:
Yes exactly like that 👍🏼

Reply:
lol, I see you put that in the issue comment already and I foolishly missed it.  Might be worth considering adding to the docs as an example